openapi/advertising/advertising.yaml

```diff
@@ -0,0 +1,811 @@
+      responses:
+        '200':
+          description: OK
+      description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
+      tags:
+        - Options
+    get:
+      summary: List Connected Accounts
+      operationId: get-connectedAccounts
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/connectedAccounts'
+                  links:
+                    type: object
+                    required:
+                      - self
+                    properties:
+                      self:
+                        type: string
+                required:
+                  - data
+                  - links
+      description: Retrieve all connected advertising accounts.
+      x-lifecycle:
+        status: trustedTester
+      security:
+        - OAuth2:
+            - advertising
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Content-Type'
+        - $ref: '#/components/parameters/businessLocation.id'
+        - $ref: '#/components/parameters/provider'
+        - $ref: '#/components/parameters/connectedAccount.resourceId'
+        - $ref: '#/components/parameters/pageLimit'
+        - $ref: '#/components/parameters/pageCursor'
+      tags:
+        - Connected Accounts
+  /campaignInfo:
+    get:
+      summary: List Campaign Info
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/campaignInfo'
+                  links:
+                    type: object
+                    required:
+                      - self
+                    properties:
+                      self:
+                        type: string
+                required:
+                  - data
+                  - links
+      operationId: get-campaignInfo
+      description: Retrieve information about Ad Campaigns under a connected advertising account.
+      tags:
+        - Campaign Info
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Content-Type'
+        - schema:
+            type: boolean
+          in: query
+          name: 'filter[isConnected]'
+          description: Filters results to fetch only connected or not connected campaigns
```

Comment:
I thought we discussed using an enum instead of a boolean so that we could accept both states at the same time.

Reply:
Whether or not a campaign is connected is either true or false.  By default the endpoint will return all, but this filter can be used to get only one or the other.